composer.json

```diff
@@ -39,7 +40,9 @@
         "symfony/yaml": "^5"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.3"
+        "phpunit/phpunit": "^8.3",
+        "slevomat/coding-standard": "~6.0",
```

Comment:
What about using `^` as in phpunit?

Reply:
Sure. Both `slevomat/coding-standard` and `squizlabs/php_codesniffer` are now using `^`-based selector.